.github/issue_label_bot.yaml

```diff
@@ -1,4 +1,5 @@
 label-alias:
   bug: 'kind/bug'
+  feature: 'kind/feature'
```

Comment:
I'm not sure this is going to work -- has the label been changed to `feature`?    The model should not have a different set of labels on this one?

Reply:
@hamelsmu Kubeflow has always used the label kind/feature. At least in the new code labels are applied in postprocessing step.
https://github.com/kubeflow/code-intelligence/blob/09c9b434fc82051f77e5d978c143a9e8e8f4a407/py/label_microservice/worker.py#L242

So if a model returns "feature" the alias would be applied but if it returns some other label it would not.

This should be what we as kubeflow want because whatever label a model is using for "feature" we want to map that to a GitHub label of "kind/feature".